### ISSUE TITLE
setup.py test doesn't seem to work

### ISSUE THREAD

**Issue by csm10495** [pdoc3/pdoc#291]
### Expected Behavior

```
python setup.py test
```

should work as shown here: https://github.com/pdoc3/pdoc/blob/master/CONTRIBUTING.md

### Actual Behavior

Python 3.8.5 on Windows, commit: bec63077a03b2

```
C:\Users\csm10495\Desktop\pdoc>python setup.py test
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running egg_info
writing pdoc3.egg-info\PKG-INFO
writing dependency_links to pdoc3.egg-info\dependency_links.txt
writing entry points to pdoc3.egg-info\entry_points.txt
writing requirements to pdoc3.egg-info\requires.txt
writing top-level names to pdoc3.egg-info\top_level.txt
reading manifest template 'MANIFEST.in'
writing manifest file 'pdoc3.egg-info\SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "setup.py", line 15, in <module>
    setup(
  File "C:\Python38\lib\site-packages\setuptools\__init__.py", line 163, in setup
    return distutils.core.setup(**attrs)
  File "C:\Python38\lib\distutils\core.py", line 148, in setup
    dist.run_commands()
  File "C:\Python38\lib\distutils\dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "C:\Python38\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "C:\Python38\lib\site-packages\setuptools\command\test.py", line 238, in run
    self.run_tests()
  File "C:\Python38\lib\site-packages\setuptools\command\test.py", line 256, in run_tests
    test = unittest.main(
  File "C:\Python38\lib\unittest\main.py", line 100, in __init__
    self.parseArgs(argv)
  File "C:\Python38\lib\unittest\main.py", line 147, in parseArgs
    self.createTests()
  File "C:\Python38\lib\unittest\main.py", line 158, in createTests
    self.test = self.testLoader.loadTestsFromNames(self.testNames,
  File "C:\Python38\lib\unittest\loader.py", line 220, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "C:\Python38\lib\unittest\loader.py", line 220, in <listcomp>
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "C:\Python38\lib\unittest\loader.py", line 205, in loadTestsFromName
    test = obj()
  File "c:\users\csm10495\desktop\pdoc\setup.py", line 11, in _discover_tests
    return unittest.defaultTestLoader.discover('pdoc.test')
  File "C:\Python38\lib\unittest\loader.py", line 346, in discover
    raise ImportError('Start directory is not importable: %r' % start_dir)
ImportError: Start directory is not importable: 'pdoc.test'

```

### Steps to Reproduce

1. Clone repo
2. pip install -e .
3. python setup.py test

### Additional info

- pdoc version: bec63077a03b2

**Comment by csm10495** [comment]
Tried something different, still didn't work:

```
C:\Users\csm10495\Desktop\pdoc>python -m unittest -v pdoc.test
test (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: test (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test
Traceback (most recent call last):
  File "C:\Users\csm10495\Desktop\pdoc\pdoc\__init__.py", line 224, in import_module
    module = importlib.import_module(module_path)
  File "C:\Python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\csm10495\Desktop\pdoc\pdoc\test\example_pkg\index.py", line 1, in <module>
    __init__.py
NameError: name '__init__' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Python38\lib\unittest\loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "C:\Users\csm10495\Desktop\pdoc\pdoc\test\__init__.py", line 39, in <module>
    EXAMPLE_PDOC_MODULE = pdoc.Module(EXAMPLE_MODULE, context=pdoc.Context())
  File "C:\Users\csm10495\Desktop\pdoc\pdoc\__init__.py", line 700, in __init__
    m = Module(import_module(fullname),
  File "C:\Users\csm10495\Desktop\pdoc\pdoc\__init__.py", line 226, in import_module
    raise ImportError(f'Error importing {module!r}: {e.__class__.__name__}: {e}')
ImportError: Error importing 'example_pkg.index': NameError: name '__init__' is not defined


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
```

**Comment by kernc** [comment]
Somewhat odd given that, as you've established, this same command is (and the former was) used to run tests in CI successfully.
https://github.com/pdoc3/pdoc/blob/bec63077a03b2136e6982825d327073ed76cc20a/.github/workflows/ci.yml#L69

I wonder if it's something to do with example modules all symbolic-linking to *\_\_init\_\_.py* module. :thinking: 
```
% tree pdoc/test/example_pkg -L 1
pdoc/test/example_pkg
├── _exclude_dir
├── _namespace
├── _private
├── __pycache__
├── _relative_import
├── _resolve_typing_forwardrefs
├── _reST_include
├── _skip_errors
├── subpkg
├── subpkg2
├── _test_linking
├── _imported_once.py
├── index.py -> __init__.py
├── __init__.py
└── module.py -> __init__.py
```

**Comment by csm10495** [comment]
Thank you. Yep its symlinks. On Windows you need to enable (Windows') Developer mode and set something in the git config. Then a fresh clone got them. Some tests are failing due to some pathing things. I'll fix that in #292. Closing.
